source/getting-started/configuration.markdown

```diff
@@ -24,20 +24,39 @@ Under the "Core" section you will find HASS Configurator.
  - Now start the add-on by clicking on START.
  - Open the user interface by clicking on OPEN WEB UI.
 
-Now let's make a small change using the configurator: we are going to change the name and location of your Home Assistant installation.
+Now let's make a change using the configurator: we are going to change the name, location, unit system, and time zone of your Home Assistant installation.
 
  - Click the folder icon in the top left of the configurator window to open the file browser sidebar.
  - Click the `configuration.yaml` file (in the `/config/` folder) to load it into the main Configurator edit window.
- - Find the `homeassistant:` configuration block, which should be the first thing in `configuration.yaml`. In this block, update `name`, `latitude`, `longitude`, `unit_system` and `time_zone` to match yours.
+ - Add the following to this file (preferably at the very top, but it ultimately doesn't matter):
+ ```yaml
+     homeassistant:
+       name: Home
+       latitude: xx.xxxx
+       longitude: xx.xxxx
+       unit_system: imperial
+       time_zone: America/Chicago
+  ```
+<div class='note'>
+ 
+  Valid options for `unit_system` are `imperial` or `metric`. See [here](https://timezonedb.com/time-zones) for a list of valid time zones. Enter the appropriate option found under the Time Zone column at that page.
+
+</div>
+
  - Click the save icon in the top right to commit changes.
- - Most changes in `configuration.yaml` require Home Assistant to be restarted to see the changes. You can verify that your changes are acceptable by running a config check. Do this by clicking on Configuration in the sidebar, click on "Server Control" and click on the "CHECK CONFIG" button. When it's valid, it will show the text "Configuration valid!".
- - Now Restart Home Assistant using the "restart" in the Server management section on the same page. In order for "Check Config" to be visible, you must enable "Advanced Mode" on your user profile.
+ - Most changes in `configuration.yaml` require Home Assistant to be restarted to see the changes. You can verify that your changes are acceptable by running a config check. Do this by clicking on Configuration in the sidebar, click on "Server Control" and click on the CHECK CONFIG button. When it's valid, it will show the text "Configuration valid!"
+ - Now Restart Home Assistant using the RESTART button in the Server management section on the same page. In order for "Check Config" to be visible, you must enable "Advanced Mode" on your user profile.
 
 <p class='img'>
 <img src='/images/screenshots/configuration-validation.png' />
 Screenshot of the "General" page in the configuration panel.
 </p>
 
+<div class='note'>
```

Comment:
Insert an empty line after this, to be sure our Markdown engine pick up the markdown inside this HTML section.